temporalio/temporalio.gemspec

```
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+require_relative 'lib/temporalio/version'
+
+Gem::Specification.new do |spec|
+  spec.name = 'temporalio'
+  spec.version = Temporalio::VERSION
+  spec.authors = ['Chad Retz']
+  spec.email = ['chad@temporal.io']
```

Fix this and remove my stuff (leftover from side Ruby work I was doing)

```suggestion
  spec.authors = ['Temporal Technologies Inc']
  spec.email = ['sdk@temporal.io']
```

---

Fixed